webrtc.html

```diff
@@ -8224,10 +8230,44 @@ <h3><dfn>RTCSctpTransport</dfn> Interface</h3>
             <var>initialState</var>, if provided, otherwise
             <code>"new"</code>.</p>
           </li>
+          <li>
+            <p>Let <var>transport</var> have a <dfn>[[\MaxMessageSize]]</dfn>
+            internal slot and run the steps labeled <a>update the data max
+            message size</a> to initialize it.</p>
+          </li>
           <li>
             <p>Return <var>transport</var>.</p>
           </li>
         </ol>
+        <p>To <dfn>update the data max message size</dfn> of an <code>
+        <a>RTCSctpTransport</a></code> run the following
+        steps:</p>
+        <ol>
+          <li>
+            <p>Let <var>transport</var> be the <code><a>RTCSctpTransport</a>
+            </code> object to be updated.</p>
+          </li>
+          <li>
+            <p>Let <var>remoteMaxMessageSize</var> be the value of the
+            "max-message-size" SDP attribute read from the remote description,
+            as described in [[!SCTP-SDP]] (section 6), or 65535 if the
+            attribute is missing.</p>
+          </li>
+          <li>
+            <p>Let <var>canSendSize</var> be the number of bytes that this
+            client can send (i.e. the size of the local send buffer) or 0 if
+            the implementation can handle messages of any size.</p>
+          </li>
+          <li>
+            <p>If either <var>remoteMaxMessageSize</var> or
+            <var>canSendSize</var> is 0, set <a>[[\MaxMessageSize]]</a> to the
+            larger of the two.</p>
```

Comment:
Although this is algorithmically fine for the *both `0` case* as the result of `max(0, 0) == 0` is correct, it might be a bit confusing to the reader.